etl/transform.py

```diff
@@ -84,6 +86,8 @@ def transform(self) -> DataFrame:
                 "source",
                 "instance_of",
                 "found_on",
+                "page_namespace",
```

Comment:
Minor nit: this select is become a bit verbose, and is duplicated. 

Not a blocker for this PR, but would you consider a minor refactoring to store column names in a list, so we can re-use it across queries?

Reply:
Same from above